### DISCUSSION TITLE
Custom markers on the nodes of the Moon

### DISCUSSION THREAD

**Discussion by kishorekumar62** [Stellarium/stellarium#3859] [Q&A] [answered]
I want to draw a pair of custom markers on the nodes of the Moon. Is there any easy way to do this? Also, I would like to have these markers stay on the nodes as I play around with time, i.e. I would like these markers to move as I speed-up time. Can this be done?

**Comment by alex-w** [comment, marked as answer]
> I want to draw a pair of custom markers on the nodes of the Moon. Is there any easy way to do this? 

Enable displaying Moon's orbit and ecliptic, clicks for custom markes (see Help window to help)

> Also, I would like to have these markers stay on the nodes as I play around with time, i.e. I would like these markers to move as I speed-up time. Can this be done?

No, it's not possible to do by custom markers

**Reply by kishorekumar62** [reply]
Thank you!
